### PR TITLE
fix: #1743 css issue on helper

### PR DESCRIPTION
**Pull request by tanphamhaiduong**
# Pull request checklist

## Does this close any currently open issues?

fixes: #1743 

**Please check if your PR fulfills the following requirements:**

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #1743 

## What is the new behavior?

- Bug fixing
- Refactor helper guide

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/elements/src/components/HelpGuide/__styles__/styles.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { css } from 'linaria'
 
-export const fadeStyle = css`
+export const helpGuildeContentStyle = css`
```
Comment (phmngocnghia) [inline packages/elements/src/components/HelpGuide/__styles__/styles.ts:3]: Can we think about the  unique class name convention? Some end with style, while some end when component name?

Comment (willmcvay) [reply packages/elements/src/components/HelpGuide/__styles__/styles.ts:3]: I agree with @phmngocnghia on this one. We should decide, happy for someone to propose something. Remember also, when we output the style it gets dashed to make it developer friendly eg help-guide-content-style.

I would suggest `<<component-name>>-<<block>>-<<modifier>>` would work.

Also you have a typo - Guide not Guilde!

File: packages/elements/src/components/HelpGuide/index.tsx [hunk 2]
```diff
@@ -90,36 +90,26 @@ export const HelpGuide = ({ children, current = 0, isLoading = false }: HelpGuid
 
   return (
     <HelpGuideContextProvider value={value}>
-      <FlexContainerBasic hasPadding flexColumn={isMobileScreen}>
+      <FlexContainerBasic className={helpGuideContainer} hasBackground hasPadding flexColumn={isMobileScreen}>
         {renderTimeline({ total, currentStep, isMobileScreen, goTo })}
-        <div className="helpguide">
```
Comment (willmcvay) [inline packages/elements/src/components/HelpGuide/index.tsx:95]: I have deleted most of this markup in my branch too! So confusing...

File: packages/elements/src/styles/components/help-guide.scss [hunk 3]
```diff
@@ -1,61 +1,6 @@
 @import '../base/colors.scss';
 @import '../base/browser.scss';
 
-.helpguide {
```
Comment (willmcvay) [inline packages/elements/src/styles/components/help-guide.scss:4]: 👍

**Pull request merged**